cogs/channel_mods.py

```diff
@@ -853,16 +853,16 @@ async def modlog(self, ctx, id_in):
 
     @modlog.command(name='delete', aliases=['del'])
     @hf.is_admin()
-    async def modlog_delete(self, ctx, user, *, index):
+    async def modlog_delete(self, ctx, user, *, indices=""):
         """Delete modlog entries.  Do `;modlog delete user -all` to clear all warnings from a user.
         Alias: `;ml del`"""
+        if not indices:  # If no index is given:
+            await hf.safe_send(ctx, "Please write numeric indices equal to or greater than 1 or "
+                                    "`-all` to clear the user's modlog.")
+            return
 
-        indexes = index.split()  # Creates a list from the prompted argument of the command.
-        not_duplicated = indexes.copy()  # Removing duplicates from the list.
-        indexes = []
-        for i in not_duplicated:
-            if i not in indexes:
-                indexes.append(i)
+        indices = indices.split()  # Creates a list from the prompted argument of the command.
+        indices = list(set(indices))  # Removing duplicates from the list.
```

Comment:
A set is like a list but it can't have duplicate entries

Reply:
Also before you had "index" as a string and "indices" as a list. I renamed the command argument to "indices" and deleted "index" then did `indices = indices.split()` so now we only have a list